### PR TITLE
Geographic coverage

### PR DESCRIPTION
**Pull request by lshepstone**
@benlovell @alexwllms @jongilbraith

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -121,19 +121,24 @@ en:
...
-        coda: in order to meet customers.
-        covers_whole_of_uk_html: 'Our firm (or this adviser specifically) is able to provide UK wide coverage.'
-
       advisers_details:
         heading: Enter the Adviser’s FCA individual reference number.
         description: You will then be asked to complete some basic information about that Adviser.
         label: Adviser FCA individual reference number *
 
+      geographical_coverage:
+        heading: Where does your adviser provide their services?
```
Comment (benlovell) [inline config/locales/en.yml]: Don't forget the `*` to denote the mandatory field. :+1:

File: config/locales/en.yml [hunk 2]
```diff
@@ -121,19 +121,24 @@ en:
...
+        description: This is to show customers if your adviser is willing to travel to provide their services.
+        covers_whole_of_uk: Does your adviser provide UK wide coverage?
+        adviser_services_postcode: Please provide details of the area your adviser offers their services?
+        postcode: Adviser's postcode
+        travel_distance: Distance the adviser is willing to travel
+        unit: Miles
+
+      advisers_qualifications:
+        heading: What are your advisers details?
+        description: You will then be asked to complete some basic information about that Adviser.
```
Comment (benlovell) [inline config/locales/en.yml]: Remember to not titleize the `Adviser` so it's consistent.

File: spec/support/adviser_page.rb [hunk 3]
```diff
@@ -8,6 +8,9 @@ class AdviserPage < SitePrism::Page
   element :confirmed_disclaimer, '.t-confirmed-disclaimer'
   element :submit, '.t-submit'
 
+  element :does_cover_whole_of_uk, '#adviser_covers_whole_of_uk_true'
```
Comment (benlovell) [inline spec/support/adviser_page.rb]: Change this to `national_coverage` and the next one to `local_coverage`

**Comment by alexwllms** [comment]
:+1:

**Comment by jongilbraith** [comment]
:+1:

**Pull request merged**